CONTRIBUTING.md

```diff
@@ -88,6 +88,15 @@ bin)/gulp` instead of `gulp`.)
 Note that running the tests also requires an installation of
 Chrome v59 or higher (v60 if you're on Windows).
 
+If you want to run a single test file, run `npm run mocha ${path/to/spec-file}`,
+substituting the actual path to the spec. Only javascript files can be executed by the `mocha` runner,
+and only those js files in the `spec` directory ending with a `.spec.js`.
+
+Alternatively, you can add an `.only` to a `describe` or `it` block (i.e. describe.only('my spec'))
```

Comment:
@el-mapache Should describe.only('my spec')) be in code ticks ie `describe.only('my spec'))`?